### PR TITLE
updated translations spanish, catalan, italian.

### LINKED ISSUE #412: update translations
feel free to update translations from spanish, catalan and italian.

find them here: https://github.com/j0rgeSD/croodle

best.

### PR DESCRIPTION
**Pull request by jelhan**
The updated translations are provided by @j0rgeSD.

Closes #412

### CODE REVIEW

**Comment by jelhan** [comment]
@vmaffione I would appreciate if you can review the updated Italian translations if you have time. I don't speak any word italian myself and you had provided some improvements in #317.

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:8]: Sì

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:63]: Devi indicare almeno un orario valido per ciascun giorno. Se hai impostato un solo giorno, devi indicare un minimo di due orari validi.

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:86]: Decifratura fallita

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:87]: La decifratura è fallita. Questo errore è probabilmente dovuto ad una chiave crittografica non corretta. Controlla la URL che hai inserito, ed in particolare la parte che segue il punto interrogativo.

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:90]: Questo potrebbe essere accaduto per diversi motivi:

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:97]: Si è verificato un errore inaspettato.

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:98]: Siamo spiacenti, ma si è verificato un errore inaspettato. Per favore, riprova più tardi.

File: translations/it.yml
Comment (vmaffione) [inline translations/it.yml:212]: supportati

Sorry, I don't get the meaning of "Partially times", so I don't know what I'm supposed to translate. What are we talking about here?

**Review by vmaffione — CHANGES_REQUESTED** [review]
Here you go :)

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:212]: Some browsers allow the user to enter a time only partially. E.g. the user enters the hour but not the minute. This validation error is shown in that case.

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:8]:
```suggestion
        label: Sì
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:63]:
```suggestion
            notEnoughTimes: 'Devi indicare almeno un orario valido per ciascun giorno. Se hai impostato un solo giorno, devi indicare un minimo di due orari validi.'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:86]:
```suggestion
            title: 'Decifratura fallita.'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:87]:
```suggestion
            description: 'La decifratura è fallita. Questo errore è probabilmente dovuto ad una chiave crittografica non corretta. Controlla la URL che hai inserito, ed in particolare la parte che segue il punto interrogativo.'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:90]:
```suggestion
            description: 'Il sondaggio che stai cercando non esiste. Questo potrebbe essere accaduto per diversi motivi:'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:97]:
```suggestion
            title: 'Si è verificato un errore inaspettato.'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:98]:
```suggestion
            description: 'Siamo spiacenti, ma si è verificato un errore inaspettato. Per favore, riprova più tardi.'
```

File: translations/it.yml
Comment (jelhan) [reply translations/it.yml:212]:
```suggestion
    timeNotPartially: "L'orario deve specificare sia le ore che i minuti."
```

**Review by jelhan — COMMENTED** [review]
@vmaffione Thanks a lot for your quick review! I converted your feedback to GitHub code suggestions. This makes it easier to review and it could be applied directly from the web interface. The feature is described [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) as point 6.

@j0rgeSD Would be awesome if you can have a look at the feedback provided.

**Comment by jelhan** [comment]
@j0rgeSD I'm currently in the process of setting up [Weblate](https://weblate.org/) to ease translating Croodle. I would like to merge this one before doing the switch to avoid merge conflicts. I hope it's okay if I merge this one with the suggestions provided by @vmaffione soon.

**Pull request merged**
